tsl/test/sql/cagg_on_cagg.sql

```diff
@@ -7,6 +7,7 @@
 \set IS_TIME_DIMENSION_WITH_TIMEZONE_1ST FALSE
 \set IS_TIME_DIMENSION_WITH_TIMEZONE_2TH FALSE
 \set IS_JOIN FALSE
+\set INTERVAL_TEST FALSE
```

Comment:
Why do we need this flag??

Reply:
Because for testing fix, I need different input in the table and caggs, also need to create extra cagg for that, but i do not want to change a whole lot of test outputs. Please let me know if there is a better way in mind...!

Reply:
Ok I got it...